script/check-new-links.sh

```diff
@@ -18,9 +20,11 @@ SKIP_PATTERNS=(
 
 if [ "_${VERBOSE}_" == "__" ]; then VERBOSE=0; fi
 
```

Comment:
Additionally the `"Run actions/checkout@v4"` step, seems to only shallow fetch the current branch, so `main` is not persent locally. So we may need to add a line like this:

```suggestion
git fetch --depth=1 origin main
```